samples/name_domains_values/Coded_Domain_HostileOrEnemy.csv

```diff
@@ -0,0 +1,2 @@
+Name,Value
+Hostile (Enemy),ENY
```

Comment:
I added this for completeness.  In each case the attribute values documented in the standard, for each of these types of text amplifiers, consist of a finite list of values.  The "N" amplifier in table VII is documented as having a display length of 3 and, if present, HAS to be "ENY".  So it seemed sensible that a user should have only two choices for the "N - Hostile(Enemy)" attribute, if it is used in a GDB, an empty string or "ENY".